app/javascript/src/components/Dashboard/Contacts/Table.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { Table as ContactTable } from "neetoui";
```

Comment:
Use most appropriate keyword. Here you are importing from `neetoui`. Thus `NeetoUI` is the best keyword. Read: https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Importing%20under%20an%20alias
```suggestion
import { Table as NeetoUITable } from "neetoui";
```